src/CoreWCF.Primitives/src/CoreWCF/Description/DispatcherBuilder.cs

```diff
@@ -222,7 +224,7 @@ internal static void InitializeServiceHost(ServiceHostBase serviceHost, IService
                     ServiceEndpoint endpoint = stuff.Value.Endpoints[i];
 
                     //EndpointFilterProvider provider = new EndpointFilterProvider();
-                    EndpointDispatcher dispatcher = BuildEndpointDispatcher(description, endpoint);
+                    EndpointDispatcher dispatcher = BuildEndpointDispatcher(description, endpoint, services);
 
```

Comment:
ServiceDescription holds a reference to IServiceProvider so no need to add a parameter to pass it through.

Reply:
done in https://github.com/CoreWCF/CoreWCF/pull/850/commits/4961f2aef7d575f7549fb95bb44bc60147747326